utils/hosted_clusters_chaos_monkey.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+trap '_delete_clusters ${EXECUTION_SEED}' SIGINT SIGTERM
+
+### EDIT VARIABLES
+LOG_FILE="/tmp/chaos_monkey.log"
+NUMBER_OF_CLUSTERS=3
+CLUSTER_NAME_SEED="chaos"
+AWS_SUBNETS="subnet-xxxxxxx,subnet-xxxxxx"
+PROVISION_SHARD="provision_shard_id:xxxxxx"
+export AWS_REGION="us-east-2"
+###
+
+exec 3>&1 1>>${LOG_FILE} 2>&1
+
+log() {
+  message=$1
+  timestamp=$(date +%Y-%m-%d\ %H:%M:%S)
+  echo -e "$timestamp\t$message"
+}
+
+_delete_clusters() {
+  log "INFO: Captured Control-C key, deleting all clusters with ${1} seed" | tee /dev/fd/3
+  for CLUSTER in $(rosa list clusters | grep $1 | awk '{print $1}'); do
+    log "INFO: Deleting cluster ${CLUSTER}" | tee /dev/fd/3
+    _delete_cluster "${CLUSTER}"
+    log "INFO: Cluster ${CLUSTER} deleted" | tee /dev/fd/3
+  done
+  exit 0
+}
+
+_random_time(){
+# Generate random number between 30 and 120
+  echo $((30 + RANDOM % (120 - 30 + 1)))
+}
+
+_create_cluster() {
+  CLUSTER_NAME=$1-$((RANDOM%10000))
+  log "INFO: Creating cluster ${CLUSTER_NAME}" | tee /dev/fd/3
+  rosa create cluster -c "${CLUSTER_NAME}" --replicas 2 --hosted-cp --sts --mode auto -y --subnet-ids $2 --properties $3 --compute-machine-type m5.xlarge --channel-group candidate --version 4.12.0-rc.4
+  rosa logs install -c "${CLUSTER_NAME}" --watch
+  log "INFO: Cluster ${CLUSTER_NAME} created" 3>&1 1>>${LOG_FILE} | tee /dev/fd/3
+}
+
+_delete_cluster(){
+  rosa delete cluster -c $1 -y
+  rosa logs uninstall -c $1 --watch
```

Comment:
Watching this means we are forced to wait for the uninstall to complete. I realize we will eventually get to a point where there may be no clusters in Ready state to remove but we should just wait then instead of waiting this entire time for the uninstall